### ISSUE TITLE
Pericardial Friction Rub

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9794]
**Preferred term label:**
Pericardial Friction Rub

**Synonyms**
Cardiac rub

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/books/NBK542284/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pericarditis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

added as

HP:0034788
